### PR TITLE
AKening

### PR DESCRIPTION
**Pull request by shazbot194**
## About The Pull Request

This pull adds in the rest of the AKs that were meant to be added into the orginal pull, as well as updating some of the stats on the Krinkov to make it more unique. 
unknown-32

Serb: Now has minor AP bonus to make it more unique. Also fixes small sprite issue with folded on suit sprites.
IH: A PD style AK that is the same as FS, but with a folding stock
Makeshift: A not great AK that can be made by just about anyone.
FS: Any randomly spawned FS AK has a random chance to have one of 3 versions, all of which are identical.
## Why It's Good For The Game

It helps flesh out some of the AKs, and gives some more variety, as well it also adds in another makeshift weapon that is a bit more convenient while still not being great. 
## Changelog
:cl: Shazbot
add: Added IH, 2 new FS AKs, and makeshift
balance: Minor tweaks to SA AK size and AP
refactor: AK icons are now in separate files
/:cl:

### CODE REVIEW

**Comment by shazbot194** [comment]
Great, it has issues that I will fix when I get home.

**Comment by shazbot194** [comment]
Should be fixed without any issues now

File: code/modules/projectiles/guns/projectile/automatic/ak47.dm [hunk 1]
```diff
@@ -61,15 +58,57 @@
...
 /obj/item/weapon/gun/projectile/automatic/ak47/fs
 	name = "FS AR .30 \"Vipr\""						// Vipr like a play on Viper and Vepr
 	desc = "Weapon of the oppressed, oppressors, and extremists of all flavours. \
 			This is a copy of an ancient semi-automatic rifle chambered for .30 Rifle. If it won't fire, percussive maintenance should get it working again. \
 			It is known for its easy maintenance, and low price. This gun is not in active military service anymore, but has become ubiquitous among criminals and insurgents. \
-			This copy has been made to look as least miltaristic as possible to be sold to as many civilian populations as possible."
-	icon_state = "AK_wood"
-	item_suffix = "_wood"
-	wielded_item_state = "_doble_wood"
+			This cheap copy has been made to look as least miltaristic as possible to be sold to as many civilian populations as possible."
```
Comment (Firefox13) [inline code/modules/projectiles/guns/projectile/automatic/ak47.dm:110]:
```suggestion
			This cheap copy has been made to look as least militaristic as possible to be sold to as many civilian populations as possible."
```

File: code/modules/projectiles/guns/projectile/automatic/ak47.dm [hunk 2]
```diff
@@ -127,3 +227,25 @@
...
+	desc = "Weapon of the oppressed, oppressors, and extremists of all flavours. \
+			This is a copy of an ancient semi-automatic rifle chambered for .30 Rifle. If it won't fire, percussive maintenance should get it working again. \
+			It is known for its easy maintenance, and low price. This gun is not in active military service anymore, but has become ubiquitous among criminals and insurgents. \
+			This crude copy shows just how forgiving the design can be."
+	icon = 'icons/obj/guns/projectile/ak/kalash.dmi'
+	w_class = ITEM_SIZE_HUGE
+	recoil_buildup = 1.6	//Full size, but cheap
+
+	origin_tech = list(TECH_COMBAT = 2)	//bad copies don't give good science
+	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_WOOD = 10)
```
[the remark this answers is not in the document]
Comment (shazbot194) [reply code/modules/projectiles/guns/projectile/automatic/ak47.dm:244]: This isn't the full material list, that is found in crafting. This is what you get from breaking it down.

File: code/modules/projectiles/guns/projectile/automatic/ak47.dm [hunk 3]
```diff
@@ -127,3 +227,25 @@
...
+
+	init_firemodes = list(
+		SEMI_AUTO_NODELAY	//too poorly made for burst or automatic
+	)
+
+	price_tag = 1000
```
[the remark this answers is not in the document]
Comment (shazbot194) [reply code/modules/projectiles/guns/projectile/automatic/ak47.dm:250]: See above but I can lower it.

File: code/modules/projectiles/guns/projectile/automatic/ak47.dm [hunk 4]
```diff
@@ -127,3 +227,25 @@
...
+	spawn_blacklisted = FALSE
```
[the remark this answers is not in the document]
Comment (shazbot194) [reply code/modules/projectiles/guns/projectile/automatic/ak47.dm:251]: Can do!

File: code/modules/projectiles/guns/projectile/automatic/ak47.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (shazbot194) [reply code/modules/projectiles/guns/projectile/automatic/ak47.dm:244]: Oh, I am dumb from just waking up. I will look at what other makeshifts are made of.

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**